helm/templates/receive/deployment.yaml

```diff
@@ -33,11 +33,14 @@ spec:
             - start
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
-            - name: http
```

Comment:
this seemed like poor naming/assignment as HTTP is over TCP and websockets are typically just run on standard 80/443. swapped them for clarity.

Reply:
makes sense to me